src/psetv2/input.js

```diff
@@ -620,7 +620,13 @@ class PsetInput {
     return this;
   }
   hasIssuance() {
-    return this.issuanceValue > 0 || this.issuanceInflationKeys > 0;
+    if (!this.issuanceBlindingNonce) {
+      return false;
+    }
+    return (
+      this.issuanceBlindingNonce.equals(transaction_1.ZERO) &&
+      (this.issuanceValue > 0 || this.issuanceInflationKeys > 0)
+    );
```

Comment:
```suggestion
    return this.issuanceBlindingNonce.equals(transaction_1.ZERO);
```

Validation checks for the issuance should be done in `input.sanityCheck()`

Reply:
can we commit suggestion?